specification/cognitiveservices/Language.Conversations/common.tsp

```diff
@@ -0,0 +1,292 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+@doc("The language service API is a suite of natural language processing (NLP) skills built with best-in-class Microsoft machine learning algorithms.  The API can be used to analyze unstructured text for tasks such as sentiment analysis, key phrase extraction, language detection and question answering. Further documentation can be found in <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview\">https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/overview</a>.")
+@service({
+    title: "Microsoft Cognitive Language Service",
+    version: "2022-05-01",
+})
+namespace Azure.AI.Language.Conversations;
+
+@doc("Error Response")
```

Comment:
See https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/data-types#Azure.Core.Foundations.Error and related. Use those types, which *should* map to these types already, but double check.